alembic/versions/5f9ad1c6faa2_migration_message.py

```diff
@@ -0,0 +1,189 @@
+"""Migration message
```

Comment:
You should generate only one new alembic file for your version upgrade with a more descriptive message

Reply:
Hi Bonifase, how can I delete an alembic version upgrade @Bonifase